contributions/presentation/week2/hheyman-nacca/README.md

```diff
@@ -0,0 +1,21 @@
+# Presentation Proposal
+
+ ## Members
+ Hugo Heyman (hheyman@kth.se)
+ Github username: [hallon-heyman](https://github.com/hallon-heyman)
+
+ Patricia Naccachian (nacca@kth.se)
+ Github username: [patricia-naccachian](https://github.com/patricia-naccachian)
+
+ ## Proposal
+The inner workings of git
+
+Our intention is to to give our viewers a deeper understanding of git beyond the more basic commands-and-branches understanding, such as the abstract object model that drives the functionality.
+ 
+ Slide draft
+ - Short history of git (also including purpose)
+ - Why git is so succesfull
+ - Branching
+ - Object-model in the deeper mechanics of git
+ - Why inner mechanics matter
+ 
```

Comment:
Your proposal is more suitable for an essay. Consider change it!

Reply:
Upon reflection I agree, however, my partner for this assignment already has another partner for her essay assignment so I would ask that we can do a good presentation on this